act-rules-format.bs

```diff
@@ -106,45 +106,68 @@ An ACT Rule MUST have a description that is in plain language, and provides a br
   </ul>
 </div>
 
-One reason why an [=atomic rule=] might not list accessibility requirements is because it is part of a [=composite rule=], and only a `failed` [=outcome=] in the composite rule would mean not satisfying the accessibility requirement. Another reason might be that an ACT Rule is designed to evaluate a best practice that is not part of any accessibility standard.
 
+Related References {#related-references}
+----------------------------------------
 
-Accessibility Requirements Mapping {#accessibility-requirements-mapping}
-------------------------------------------------------------------------
+Related references are non-normative accessibility practices, such as Advisory Techniques that enhance accessibility beyond minimum conformance. In some cases related references could imply conformance with a related accessibility requirement.
 
-If any accessibility requirements are listed, the ACT Rule MUST indicate how each outcome impacts the conformance to the listed accessibility requirements. For WCAG 2.1 Success Criteria, this means indicating whether or not the success criterion can be satisfied, or if further testing is needed.
+An ACT Rule MUST list an identifier and reference for each related reference that it is written to directly test. When related references are listed, the rule MUST indicate that these are not required for conformance. This MAY be omitted when rules are designed to be part of a [=composite rule=].
 
 <div class=example>
-  <p><strong>Example</strong>: The outcome of this rule maps to the accessibility requirements in the following manner:</p>
+  <p><strong>Example</strong>: The outcome of this rule indicates conformance beyond the related WCAG 2.1 success criteria:</p>
   <ul>
-    <li>`failed`: The success criteria are *not satisfied*</li>
-    <li>`passed` or `inapplicable`: Further testing is necessary to determine if the success criteria are satisfied</li>
+    <li>[G127: Identifying a Web page's relationship to a larger collection of Web pages](https://www.w3.org/TR/WCAG20-TECHS/G127.html)</li>
+  </ul>
+</div>
+
+
+Rule Mapping {#rule-mapping}
+----------------------------
+
+For all [accessibility requirements](#accessibility-requirements) and [related references](#related-references) listed, the ACT Rule MUST indicate how each outcome relates to the listed accessibility references. That is, the rule specifies if each accessibility reference is `satisfied`, `not satisfied`, or `undetermined` for the rule outcomes `passed`, `failed`, and `inapplicable`.
+
+Every mapping MUST be consistent with the related accessibility references. When rules refer to accessibility requirements, the mapping MUST result in either `satisfied` or `not satisfied` for any of the rule outcomes. That is, the mapping is not `undetermined` for all outcomes when mapping to accessibility requirements. Such mappings indicate related references rather than requirements.
```

Comment:
So `failed` could mean the SC is undetermined, as long as `passed` means the SC is satisfied? I'm not sure about the implications of this. That would let you write rules like "don't use iframe, video or object elements in HTML" to pass 1.2.3. I'm not sure I fully understand the implications of allowing this.

Reply:
I reworded that to be clearer. Again, I think nothing is new here. We just want to avoid having an accessibility requirement where all outcomes imply "further testing is needed". I think we agreed on that during the f2f.